spec/formatting.md

```diff
@@ -222,6 +222,71 @@ the following steps are taken:
 
    The form that resolved _operand_ and _option_ values take is implementation-defined.
 
+   A _declaration_ binds the resolved value of an _expression_
```

Comment:
Not asking for any change in the tech preview, but this will definitely need to be revised afterwards.
The whole notion of a 'resolved value' is very muddy, unless it literally means "what is bound to a variable by an expression declaration", which means it *also* includes selection/formatting options and their values whenever those are carried over.

Reply:
#645 is an attempt to address this. (But it could go even further.)